packages/big-design/src/components/Pagination/Pagination.tsx

```diff
@@ -95,13 +96,31 @@ export const Pagination: React.FC<PaginationProps> = memo(
           />
         </Flex.Item>
         <Flex.Item>
-          <StyledButton variant="subtle" disabled={currentPage <= 1} onClick={handlePageDecrease}>
-            <ChevronLeftIcon title="Previous page" />
-          </StyledButton>
+          <Button
+            iconOnly={
+              <ChevronLeftIcon
+                title="Previous page"
+                size="xLarge"
+                color={currentPage <= 1 ? 'secondary30' : 'secondary70'}
```

Comment:
What is the reason for adding these two props? As far as I can tell, you don't need them.